src/java/picard/fingerprint/HaplotypeProbabilitiesFromContaminatorSequence.java

```diff
@@ -92,7 +92,6 @@ public void addToProbs(final Snp snp, final byte base, final byte qual) {
     private void updateLikelihoods() {
         final double[] ll = new double[Genotype.values().length];
         for (final Genotype contGeno : Genotype.values()) {
-            // p(a | g_c) = \sum_g_m { P(g_m) \prod_i P(a_i| g_m, g_c)}
```

Comment:
Please don't remove this line - it's not commented out java code, it is actual documentation.